src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationIdentificationParametersPresenter.cs

```diff
@@ -32,13 +34,15 @@ public class ParameterIdentificationIdentificationParametersPresenter : Abstract
       private readonly List<IdentificationParameterDTO> _allIdentificationParameterDTOs = new List<IdentificationParameterDTO>();
       public event EventHandler<IdentificationParameterEventArgs> IdentificationParameterSelected = delegate { };
       public event EventHandler NoIdentificationParameterSelected = delegate { };
+      private readonly IEventPublisher _eventPublisher;
 
       public ParameterIdentificationIdentificationParametersPresenter(IParameterIdentificationIdentificationParametersView view, IIdentificationParameterFactory identificationParameterFactory,
-         IIdentificationParameterToIdentificationParameterDTOMapper identificationParameterDTOMapper, IIdentificationParameterTask identificationParameterTask) : base(view)
+         IIdentificationParameterToIdentificationParameterDTOMapper identificationParameterDTOMapper, IIdentificationParameterTask identificationParameterTask, IEventPublisher eventPublisher) : base(view)
```

Comment:
If you are making corrections already in this file, I've been breaking these very long parameter lists into one line per parameter to make it easier to read.

Reply:
The long list can be broken into one injected object per line so it's easier to read.